CLI/vault_actions/push.go

```diff
@@ -0,0 +1,53 @@
+package vault_actions
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+
+	vault "github.com/hashicorp/vault/api"
+)
+
+type Variable struct {
+	Key   string
+	Value string
+}
+
+func LinesInFile(fileName string) []string {
+	f, _ := os.Open(fileName)
+	scanner := bufio.NewScanner(f)
+	result := []string{}
+	for scanner.Scan() {
+		line := scanner.Text()
+		result = append(result, line)
+	}
+	return result
+}
+
+func Create_var(data string) Variable {
+	var variable Variable
+	variable.Key = strings.Split(data, "=")[0]
+	variable.Value = strings.Split(data, "=")[1]
+	variable.Value = strings.Trim(variable.Value, "\"")
+	return variable
+}
+
+func IsVariableSecureEnvLocal(variable Variable) bool {
```

Comment:
why do you add an uppercase if it's only used in this file ?
```suggestion
func isVariableSecureEnvLocal(variable Variable) bool {
```